packages/api/src/routes/medical/internal-docs.ts

```diff
@@ -95,6 +98,61 @@ router.post(
   })
 );
 
+/** ---------------------------------------------------------------------------
+ * POST /internal/docs/re-convert
```

Comment:
this comment doesn't have to be addressed now, but it a higher-level food for thought:

AFAIK there is nothing stopping us from accidentally hitting this endpoint twice, causing weird/unintended behavior

a better design for all these long-running internal operations would be to make them idempotent in the sense that if an operation is already running, no action is taken on further executions until the original op is finished (ie like we have for DQs)

perhaps we can raise a ticket for this to bake it into our architecture?

Reply:
Yeah, I know what you mean. We have a bunch of those. I thought about this a few times but never created a ticket, not this time! :) 

Here it goes:

https://github.com/metriport/metriport-internal/issues/1393